vars/ciPipeline.groovy

```diff
@@ -37,6 +37,7 @@ def call(Map parameters = [:], Closure body) {
     def postBuild = parameters.get('postBuild')
     def timeoutValue = parameters.get('timeout', 120)
     def sendMetrics = parameters.get('sendMetrics', true)
+    def bodyWrapper = parameters.get('wrapper', { Closure body -> body() })
```

Comment:
I think the compiler will complain body is defined twice here.

Reply:
Nice catch! Thank you. Fixed it.